test/libsolidity/syntaxTests/inheritance/override/nonintermediate_common_base_and_unique_implementation.sol

```diff
@@ -0,0 +1,19 @@
+interface I {
+	function f() external;
+	function g() external;
+}
+interface J {
+	function f() external;
+}
+abstract contract IJ is I, J {
+	function f() external virtual override (I, J);
+}
+abstract contract A is IJ
+{
+	function f() external override {}
```

Comment:
Can you add a similar test where this function only is not implemented?

Reply:
Will add.